libsnark/jsnark_interface/CircuitReader.cpp

```diff
@@ -13,9 +13,9 @@ CircuitReader::CircuitReader(char* arithFilepath, char* inputsFilepath,
 	numWires = 0;
 	numInputs = numNizkInputs = numOutputs = 0;
 
-	parseAndEval(arithFilepath, inputsFilepath);
-	constructCircuit(arithFilepath);
-	mapValuesToProtoboard();
+	parseAndEval(arithFilepath, inputsFilepath); // 14s
```

Comment:
nit: how do you get the timing result? (e.g. It should be related to OS, arithFile size, CPU clock rate, Disk speed, etc.) maybe it's better to document somewhere else about the performance.

Reply:
Oh my bad. This timing specifically means tlsNotary circuit (in the repo version) on my machine

Reply:
added